config/locales/claims/en/support/schools/users.yml

```diff
@@ -18,9 +18,9 @@ en:
                 first_name: First name
                 last_name: Last name
                 email: Email
```

Comment:
I think on the Prototype this is Email address 
![Screenshot from 2024-01-17 16-13-11](https://github.com/DFE-Digital/itt-mentor-services/assets/11318084/bd3071c7-d670-4a33-ac27-a22c11455c7e)

Reply:
I've started on some work that uses `User.human_attribute_name(:email)` => `Email address` for these labels. FYI :)

Reply:
Good catch @CatalinVoineag . Resolved in [f5e3dbb](https://github.com/DFE-Digital/itt-mentor-services/pull/132/commits/f5e3dbbbc8c592c86d73fdf19629fc3d0a5aaf11)